### PR TITLE
Quin

### PR DESCRIPTION
**Pull request by quinlanj**

### CODE REVIEW

File: app/src/main/java/edu/stanford/junwonpk/deeplistener/WebAlmondTestApp.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            session.close();
+
+            // wait 2 seconds for messages from websocket
+            Thread.sleep(2000);
+
+            // This thread should be kept alive for the duration of the connection
+
+
+        } catch (InterruptedException ex) {
+            System.err.println("InterruptedException exception: " + ex.getMessage());
```
Comment (mikexing2010) [inline app/src/main/java/edu/stanford/junwonpk/deeplistener/WebAlmondTestApp.java]: How will these System.err.println messages show up on the app? Maybe consider Log.e() function instead?

Comment (junwonpk) [inline app/src/main/java/edu/stanford/junwonpk/deeplistener/WebAlmondTestApp.java]: Toast can serve the purpose too

Comment (quinlanj) [inline app/src/main/java/edu/stanford/junwonpk/deeplistener/WebAlmondTestApp.java]: Ok, changed to log.e/d

**Comment by mikexing2010** [comment]
Changes look good. So far this code seems to be standalone, so it shouldn't have any adverse affect on the rest of the system. Let's get it checked in.

**Pull request merged**